MapboxDirections/MBRouteStep.swift

```diff
@@ -513,6 +513,11 @@ public class RouteStep: NSObject, NSSecureCoding {
      */
     public let destinations: String?
     
+    /**
+     Array of interersections
```

Comment:
Typo. Also, explain that these are all the intersections along the step, starting with the one at the maneuver and ending with the one just before the next step's maneuver.